src/live_restore/live_restore_fs.c

```diff
@@ -117,30 +117,29 @@ __live_restore_debug_dump_extent_list(WT_SESSION_IMPL *session, WTI_LIVE_RESTORE
 #pragma GCC diagnostic pop
 
 /*
- * __live_restore_create_tombstone_path --
- *     Generate the file path of a tombstone for a file. This tombstone does not need to exist.
+ * __live_restore_create_stop_file_path --
```

Comment:
super nit: Is it a "stop file" or a "stopfile"? I have the vaguest of preferences for the latter.

Reply:
nah 1st.